Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-buster as build
+FROM python:3.12-rc-buster as build
```

Comment:
I believe `rc` here means "release candidate". Is there not a full release available?